server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -120,6 +120,9 @@ public class RemoteClusterStateService implements Closeable {
 
     private final AtomicBoolean deleteStaleMetadataRunning = new AtomicBoolean(false);
 
+    private static final int INDEX_METADATA_CODEC_VERSION = 1;
+    private static final int MANIFEST_CODEC_VERSION = 1;
```

Comment:
you would need to handle the case of missing codec version for manifest file which would have global metadata file name missing.

Reply:
We are going to handle that in Upload PR of metadata. [https://github.com/opensearch-project/OpenSearch/pull/10404]